icons/cassette-tape.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="20" height="16" x="2" y="4" rx="2" />
+  <circle cx="8" cy="10" r="2" />
+  <path d="M8 12h8" />
+  <circle cx="16" cy="10" r="2" />
+  <path d="m6 20 .8-3c.1-.5.7-1 1.2-1h8c.6 0 1.1.4 1.2 1l.8 3" />
```

Comment:
As usual, there are some issues with curvature, shame on you, Adobe Illustrator. :(
```suggestion
  <path d="m6 20 .7-2.9A1.4 1.4 0 0 1 8.1 16h7.8a1.4 1.4 0 0 1 1.4 1l.7 3"/>
```